lxd/instance/drivers/driver_qemu.go

```diff
@@ -677,6 +677,20 @@ func (d *qemu) ovmfPath() string {
 	return "/usr/share/OVMF"
 }
 
+func killQemuProcess(d *qemu, pid int) {
```

Comment:
Could you make this a function on the qemu struct rather than standalone please. Then you would use the qemu argument as a function receiver rather than a separate argument.